src/is-my-room/index.ts

```diff
@@ -0,0 +1,9 @@
+export function isMyRoom(roomName: string): boolean{
+  return !!(
+    roomName in Game.rooms
```

Comment:
It's probably a good idea to use `@open-screeps/is-room-visible`.

Reply:
agreed, I will sort that.